GitCommands/RevisionGraph.cs

```diff
@@ -186,12 +186,15 @@ private void ProccessGitLog(CancellationToken taskState)
                 (BranchFilter.IndexOfAny(ShellGlobCharacters) >= 0))
                 branchFilter = "--branches=" + BranchFilter;
 
+            string revisionsToPathsSeparator = Filter.Contains(" -- ") ? string.Empty : " --"; // When there is a path filter it already contains the separator
```

Comment:
It seems like a kludge.
Could you place `--` between `{0} {3}` and fix `Filter` so that it didn't contain `--`?